Source/ACE/Entity/Container.cs

```diff
@@ -41,6 +42,20 @@ public Container(AceObject aceObject, ObjectGuid guid)
         }
 
         // Inventory Management Functions
+        public virtual void SaveInventory()
```

Comment:
I don't think Container is the place for this.  The containers that save to the database are special cases.  Namely, housing storage and players.  It should go in those places.